scripts/ConditionalTest.js

```diff
@@ -14,22 +14,6 @@ const ConditionalTest = {
     return process.env.JEST_CIRCUS === '1';
   },
 
-  skipSuiteOnJasmine() {
```

Comment:
we'll probably need this when we start adding tests for things that we fixed in circus but that don't work in jasmine (like nested `test` or running hooks when there's no tests)

Reply:
ok, brought it back